hummingbot/client/ui/parser.py

```diff
@@ -149,6 +150,19 @@ def load_parser(hummingbot: "HummingbotApplication", command_tabs) -> [ThrowingA
     previous_strategy_parser.add_argument("option", nargs="?", choices=["Yes,No"], default=None)
     previous_strategy_parser.set_defaults(func=hummingbot.previous_strategy)
 
+    mqtt_parser = subparsers.add_parser("mqtt", help="Manage MQTT Bridge to Message brokers")
+    mqtt_parser.add_argument("subcommand",
```

Comment:
this can be addressed with the same structure of the gateway commands

Reply:
Just merged a commit that implements the mqtt command using subparsers as proposed by @cardosofede .

https://github.com/hummingbot/hummingbot/pull/5945/commits/4456ea2837e191edb5d199959bb4ccda48a9e6be